monitoring/rid_qualifier/run_locally.sh

```diff
@@ -0,0 +1,25 @@
+
+#!/usr/bin/env bash
+
+AUTH='--auth=NoAuth()'
+# NB: A prerequisite to run this command locally is to have a running instance of the rid_qualifier/mock (/monitoring/rid_qualifier/mock/run_locally.sh)
+
+LOCALE='--locale=che'
+
+INJECTION_URL='--injection_base_url=http://localhost:8070/sp/uss1'
```

Comment:
This is a good flag to pass when running rid_qualifier from Python directly on a developer machine because, in that context, `localhost` refers to the developer machine.  However, when we run rid_qualifier in a Docker container, `localhost` then refers to the container, not the developer machine.  And, that container is not running an instance of rid_qualifier mock -- instead, the rid_qualifier mock is in a different container.  So, rid_qualifier won't be able to reach the mock as-is ("Failed to establish a connnection...").

One solution is to make sure both containers are on the same Docker network and then use the hostname of the container running the mock.  But, that takes a few setup steps -- an easier approach is to have the container simply connect to the host machine.  In Docker, the hostname of the host machine (from the perspective of a container) is `host.docker.internal`.  So, instead of having the `injection_base_url` be `http://localhost:8070/sp/uss1` as it would be if running rid_qualifier directly with the developer machine's Python, simply change it to `http://host.docker.internal:8070/sp/uss1`, and the rid_qualifier will be able to connect to the mock from inside the container it is run from in this file.

Reply:
Thanks for this! Yes I was trying number of things in Docker land. I have made the change and will test it.